Tests/DependencyInjection/SonataPageExtensionTest.php

```diff
@@ -0,0 +1,92 @@
+            'Sonata\\PageBundle\\CmsManager\\CmsManagerSelectorInterface',
+            'Sonata\\PageBundle\\CmsManager\\CmsPageManager',
+            'Sonata\\PageBundle\\CmsManager\\CmsSnapshotManager',
+            'Sonata\\PageBundle\\CmsManager\\DecoratorStrategy',
+            'Sonata\\PageBundle\\CmsManager\\DecoratorStrategyInterface',
+            'Sonata\\PageBundle\\Entity\\BaseBlock',
+            'Sonata\\PageBundle\\Entity\\BasePage',
+            'Sonata\\PageBundle\\Entity\\BaseSite',
+            'Sonata\\PageBundle\\Entity\\BaseSnapshot',
+            'Sonata\\PageBundle\\Entity\\BlockInteractor',
+            'Sonata\\PageBundle\\Entity\\BlockManager',
+            'Sonata\\PageBundle\\Entity\\PageManager',
+            'Sonata\\PageBundle\\Entity\\SiteManager',
+            'Sonata\\PageBundle\\Entity\\SnapshotManager',
+            'Sonata\\PageBundle\\Entity\\Transformer',
+            'Sonata\\PageBundle\\Generator\\Mustache',
+            'Sonata\\PageBundle\\Listener\\ExceptionListener',
+            'Sonata\\PageBundle\\Listener\\RequestListener',
+            'Sonata\\PageBundle\\Listener\\ResponseListener',
+            'Sonata\\PageBundle\\Model\\Block',
+            'Sonata\\PageBundle\\Model\\BlockInteractorInterface',
+            'Sonata\\PageBundle\\Model\\Page',
+            'Sonata\\PageBundle\\Model\\PageBlockInterface',
+            'Sonata\\PageBundle\\Model\\PageInterface',
+            'Sonata\\PageBundle\\Model\\PageManagerInterface',
+            'Sonata\\PageBundle\\Model\\Site',
+            'Sonata\\PageBundle\\Model\\SiteInterface',
+            'Sonata\\PageBundle\\Model\\SiteManagerInterface',
+            'Sonata\\PageBundle\\Model\\Snapshot',
+            'Sonata\\PageBundle\\Model\\SnapshotChildrenCollection',
+            'Sonata\\PageBundle\\Model\\SnapshotInterface',
+            'Sonata\\PageBundle\\Model\\SnapshotManagerInterface',
+            'Sonata\\PageBundle\\Model\\SnapshotPageProxy',
+            'Sonata\\PageBundle\\Model\\Template',
+            'Sonata\\PageBundle\\Page\\PageServiceManager',
+            'Sonata\\PageBundle\\Page\\PageServiceManagerInterface',
+            'Sonata\\PageBundle\\Page\\Service\\BasePageService',
+            'Sonata\\PageBundle\\Page\\Service\\DefaultPageService',
+            'Sonata\\PageBundle\\Page\\Service\\PageServiceInterface',
+            'Sonata\\PageBundle\\Page\\TemplateManager',
+            'Sonata\\PageBundle\\Page\\TemplateManagerInterface',
+            'Sonata\\PageBundle\\Request\\SiteRequestContext',
+            'Sonata\\PageBundle\\Route\\CmsPageRouter',
+            'Sonata\\PageBundle\\Site\\BaseSiteSelector',
+            'Sonata\\PageBundle\\Site\\HostPathSiteSelector',
+            'Sonata\\PageBundle\\Site\\HostSiteSelector',
+            'Sonata\\PageBundle\\Site\\SiteSelectorInterface',
+            'Sonata\\PageBundle\\Twig\\Extension\\PageExtension',
+            'Sonata\\PageBundle\\Twig\\GlobalVariables',
```

Comment:
Think about this test. How will it fail : 

1. If someone adds a new class to compile
2. If someone removes a class from compile
3. If somehow, the code is refactored and the call is made, but with a completely different argument

IMO, the only interesting case when you want it to fail because here is the reaction of the developer when this test fails : 

1. Duh, of course if fails, I added a class to compile
1. Duh, of course if fails, I removed a class from compile
1. Uh oh, this is very interesting, my refactoring is probably not good, the argument does not look like what it was before.

To have the test fail only for 3. , you should assert that the argument looks like a big array of classes (but really, what classes are in there does not matter).